pdns/packethandler.cc

```diff
@@ -512,7 +518,7 @@ void PacketHandler::doAdditionalProcessing(DNSPacket& p, std::unique_ptr<DNSPack
           if (content.isRoot()) {
             content = rr.dr.d_name;
           }
-          content = doAdditionalServiceProcessing(content, rr.dr.d_type, r);
+          content = doAdditionalServiceProcessing(content, rr.dr.d_type, r, extraRecords);
```

Comment:
```suggestion
          if (rrc->getPriority() == 0) {
            content = doAdditionalServiceProcessing(content, rr.dr.d_type, r, extraRecords);
          }
```
Only perform  AdditionalServiceProcessing for aliasform records.